CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contribution guide
```

Comment:
Point to https://equinor.github.io/dm-docs/contributing, e.g.
---

# Contribution guide

Want to contribute to dm-app-dmt? There are a few things you need to know.

We wrote a [contribution guide](https://equinor.github.io/dm-docs/contributing) to help you get started.

---
?